File: list_my_files.jl
#!/usr/bin/env julia

# Get current directory
rdir = pwd()

# List files
archivos = readdir(rdir)

# Build output file
outfile = joinpath(rdir, "list_files_withJulia.txt")

# Open to write
open(outfile, "w") do io
    write(io, "my files: \n")
    for archivo in archivos
        write(io, archivo * "\n")
    end
end

println("y con mi... hacha!")

File: list_my_files.py
#!/usr/bin/env python3

import os

# Get working directory
rdir = os.getcwd()

# List files
archivos = os.listdir(rdir)

# Open output file
outfile = os.path.join(rdir, "list_files_withPython.txt")

# Write
with open(outfile, "w", encoding="utf-8") as f:
    f.write("my files: \n")
    for archivo in archivos:
        f.write(f"{archivo}\n")

print("Cuenta con mi espada")
